mmv1/products/networkservices/LbRouteExtension.yaml

```diff
@@ -0,0 +1,177 @@
+          item_type: !ruby/object:Api::Type::NestedObject
+            properties:
+              - !ruby/object:Api::Type::String
+                name: 'name'
+                description: |
+                  The name for this extension. The name is logged as part of the HTTP request logs.
+                  The name must conform with RFC-1034, is restricted to lower-cased letters, numbers and hyphens,
+                  and can have a maximum length of 63 characters. Additionally, the first character must be a letter
+                  and the last a letter or a number.
+                required: true
+              - !ruby/object:Api::Type::String
+                name: 'authority'
+                description: |
+                  The :authority header in the gRPC request sent from Envoy to the extension service.
+              - !ruby/object:Api::Type::String
+                name: 'service'
+                description: |
+                  The reference to the service that runs the extension. Must be a reference to a backend service
+                required: true
+                diff_suppress_func: 'tpgresource.ProjectNumberDiffSuppress'
+              - !ruby/object:Api::Type::String
+                name: 'timeout'
+                description: |
+                  Specifies the timeout for each individual message on the stream. The timeout must be between 10-1000 milliseconds.
+                  A duration in seconds with up to nine fractional digits, ending with 's'. Example: "3.5s".
+                diff_suppress_func: 'tpgresource.DurationDiffSuppress'
+              - !ruby/object:Api::Type::Boolean
+                name: 'failOpen'
+                description: |
+                   Determines how the proxy behaves if the call to the extension fails or times out.
+                   When set to TRUE, request or response processing continues without error.
+                   Any subsequent extensions in the extension chain are also executed.
+                   When set to FALSE: * If response headers have not been delivered to the downstream client,
+                   a generic 500 error is returned to the client. The error response can be tailored by
+                   configuring a custom error response in the load balancer.
+              - !ruby/object:Api::Type::Array
+                name: 'forwardHeaders'
+                description: |
+                  List of the HTTP headers to forward to the extension (from the client or backend).
+                  If omitted, all headers are sent. Each element is a string indicating the header name.
+                item_type: Api::Type::String
+  - !ruby/object:Api::Type::Enum
+    name: 'loadBalancingScheme'
+    description: |
+      All backend services and forwarding rules referenced by this extension must share the same load balancing scheme.
+        For more information, refer to Choosing a load balancer.
+    values:
+      - :LOAD_BALANCING_SCHEME_UNSPECIFIED
```

Comment:
This value is not meaningless and can be removed.

Reply:
removed